src/gui/accountsettings.cpp

```diff
@@ -287,6 +287,9 @@ void AccountSettings::slotCustomContextMenuRequested(const QPoint &pos)
     bool folderPaused = _model->data(index, FolderStatusDelegate::FolderSyncPaused).toBool();
     bool folderConnected = _model->data(index, FolderStatusDelegate::FolderAccountConnected).toBool();
     auto folderMan = FolderMan::instance();
+    QPointer<Folder> folder = folderMan->folder(alias);
```

Comment:
minor: why the QPointer?

Reply:
Because I need it for #6744  .  And there we need QPointer because there will be a message box showing and it might get deleted while the message box is shown.